src/cosmology/cosmology.h

```diff
@@ -19,6 +19,9 @@ class Cosmology
   Real Omega_L;
   Real Omega_K;
   Real Omega_b;
+  Real Omega_R;
+  Real w0;
+  Real wa;
```

Comment:
I've been thinking about how best to clean this up. It will also impact the cooling and RT portions of the code, but those need to be reconciled anyway. I will come back to this in the future.